deckhand/control/api.py

```diff
@@ -15,16 +15,46 @@
 import os
 
 import falcon
+from oslo_config import cfg
+from oslo_log import log as logging
 
+from deckhand.conf import config
 from deckhand.control import base as api_base
 from deckhand.control import secrets
 
+CONF = cfg.CONF
+
+
+def __setup_logging():
```

Comment:
It is an OpenStack pattern.  I suppose one of the immediate advantages is that if your logging approach is exactly the same across all applications and they are all using `oslo_log` they can all receive the same configuration file which is now separated from the application specific configuration.  

From a legacy perspective, it also means logging configuration, which sometimes is under the per-view of operations and not application owners can be independently controlled outside the applications own configuration.

In any event, there is likely value in adopting this unified approach across the board if only to copy each others approach for managing the now log-specific logging.conf template.